### PR TITLE
Migrate XmlFileConfigStore to @configu/integrations (issue 524)

### PR DESCRIPTION
**Pull request by alfasin**
This PR solves: https://github.com/configu/configu/issues/524

Before you send over your PR, let's ensure it’s all set for a smooth review. Here’s a quick checklist:

Please verify that:
* [X] The PR has a clear title and a concise summary of the changes (Use the present tense and imperative mood for your descriptions)
* [X] Related issues are linked using `Closes #number`
* [X] Code is up-to-date with the `main` branch
* [X] There are no linting or formatting errors
* [X] There are new or updated unit tests validating the change

For more details, have a look at our CONTRIBUTING.md:
  https://github.com/configu/configu/blob/main/CONTRIBUTING.md

**Please** focus your PR on a single topic to avoid mixing unrelated changes.

Once you submit your PR, we’ll dive into it as soon as we can. We might come back with some suggestions or ask for some tweaks.

Thank you for your pull request!

---

## For Maintainers

Please ensure:
* [ ] The PR is linked to an issue and both are assigned to the same person
* [ ] The PR is labeled as either 'feat', 'bug', or 'chore'
* [ ] The changes have been approved by a reviewer
* [ ] The contributor checklist is fully addressed

### CODE REVIEW

File: packages/cli/package.json [hunk 1]
```diff
@@ -38,6 +38,7 @@
   "dependencies": {
     "@configu/sdk": "workspace:*",
     "@configu/common": "workspace:*",
+    "@configu-integrations/xml-file": "workspace:*",
```
Comment (davidmeirlevy) [inline packages/cli/package.json:41]: we don't install our integration inside the CLI.

File: packages/cli/src/commands/base.ts [hunk 2]
```diff
@@ -14,7 +15,7 @@ export abstract class BaseCommand extends Command<Context> {
     this.context.configu = configu;
 
     // register builtins here
-    Registry.register({ NoopConfigStore, InMemoryConfigStore });
+    Registry.register({ NoopConfigStore, InMemoryConfigStore, XmlFileConfigStore });
```
Comment (davidmeirlevy) [inline packages/cli/src/commands/base.ts:18]: the xml might not be one of the built-in integrations.
please remove from the CLI.

File: packages/integrations/stores/xml-file/src/configs-db.xml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
```
Comment (davidmeirlevy) [inline packages/integrations/stores/xml-file/src/configs-db.xml:1]: is that file used for testing?
if so, it's better to create a test file, and put it inside a "stub" directory. (you can see a testing structure example in (package/common)

**Review by davidmeirlevy — CHANGES_REQUESTED** [review]
Thanks for your PR!

I wrote some comments. please fix so we can merge. 🙏

**Comment by davidmeirlevy** [comment]
@alfasin thanks for the PR!
just a few minor fixed, and we can merge. 👏

File: packages/cli/src/commands/base.ts [hunk 4]
```diff
@@ -1,7 +1,8 @@
 import { Command, Option } from 'clipanion';
 import _ from 'lodash';
 import { CfguFile, ConfiguFile, parseJSON, Registry } from '@configu/common';
-import { EvalCommandOutput } from '@configu/sdk';
+import { EvalCommandOutput, NoopConfigStore, InMemoryConfigStore } from '@configu/sdk';
+import { XmlFileConfigStore } from '@configu-integrations/xml-file/src/xml-file';
```
Comment (davidmeirlevy) [inline packages/cli/src/commands/base.ts:5]: @alfasin please revert this change as well.

**Review by davidmeirlevy — CHANGES_REQUESTED** [review]

**Review by Jonathan-den — APPROVED** [review]

**Review by davidmeirlevy — APPROVED** [review]

**Comment by davidmeirlevy** [comment]
@alfasin thanks!!

**Pull request merged**